src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
+  --active-color: #00acdc;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+img {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+}
+
+.nav__item {
+  list-style: none;
+  margin: 0 10px;
+}
+
+.nav__link {
+  color: black;
+  display: flex;
+  position: relative;
+  height: 60px;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  text-decoration: none;
+  align-items: center;
+}
+
+.nav__item:first-child {
+  margin: 0 10px 0 0;
+}
+
+.nav__item:last-child {
+  margin: 0 0 0 10px;
+}
```

Comment:
it's better to use one-direction margin
![image](https://github.com/mate-academy/layout_moyo-header/assets/55200530/ae8eedbc-5c73-43e0-9390-590a60af96c3)


use right or left with value 20px

and margin 0 for first or last element

much better practice